docs/using/common.md

```diff
@@ -69,7 +69,7 @@ You do so by passing arguments to the `docker run` command.
   (The `start-notebook.sh` script will `su ${NB_USER}` after adding `${NB_USER}` to sudoers.)
   **You should only enable `sudo` if you trust the user or if the container is running on an isolated host.**
 - `-e GEN_CERT=yes` - Instructs the startup script to generates a self-signed SSL certificate and configure Jupyter Notebook to use it to accept encrypted HTTPS connections.
-- `-e JUPYTER_ENABLE_LAB=yes` - Instructs the startup script to run `jupyter lab` instead of the default `jupyter notebook` command.
+- `-e JUPYTER_CMD=yes` - Instructs the startup script to run `jupyter ${JUPYTER_CMD}` instead of the default `jupyter lab` command.
```

Comment:
`yes` should be changed to the list of available options or to some subsection of the docs.

I propose to move `Switching back to classic` from recipes to a subsection here because it's not a recipe, but a feature we support ourselves.

Reply:
It's done. I've chosen to put it in the _Alternative Commands_ section to avoid to have too much level 2 titles in the page. Let me know if you have a different opinion.